src/services/validation.schema.js

```diff
@@ -110,6 +120,7 @@ const schema = {
     ],
     oneOf: [
       { required: ["operator_company_name", "operator_company_house_number"] },
+      { required: ["operator_charity_name", "operator_charity_number"] },
```

Comment:
@Djangofs think the next time we add something to the `oneOf` setup here, we will find it much easier after a refactor/split out into separate files like we chatted about